testcases/master/template-test.jinja2

```diff
@@ -0,0 +1,9 @@
+{% extends "testcases/master/template-master.jinja2" %}
+
+{% block test_target %}
+  {{ super() }}
+    - repository: {{ TEST_DEFINITIONS_REPOSITORY }}
+      from: git
+      path: '{{test_path_file}}'
+      name: {% if test_plan_name is undefined %}'{{test_name}}'{% else %} '{{test_plan_name}}'{% endif %}
```

Comment:
this is going to break your results in qa-reports. "name" in this section defines test suite name in qa-reports.